models/incremental/fct_dbt_test_executions.sql

```diff
@@ -13,7 +13,15 @@ with tests as (
         test_name,
         test_type,
         model_schema,
-        replace(model_path, 'bespoke_tests/', '') as model_path
+        case
```

Comment:
i wonderrrr if a case when like this is any better than like three embedded replace statements, if youre doing three replace statements anyway?

like `replace(replace(replace, model_path...blah blah`